core/src/main/java/io/questdb/cairo/wal/ApplyWal2TableJob.java

```diff
@@ -378,13 +378,12 @@ private void applyOutstandingWalTransactions(
                             );
 
                             if (added > -1L) {
-                                long timeDelta = microClock.getTicks() - start;
-                                insertTimespan += timeDelta;
+                                insertTimespan += microClock.getTicks() - start;
                                 rowsAdded += added;
                                 iTransaction++;
                                 long physicallyWrittenRowsSinceLastCommit = writer.getPhysicallyWrittenRowsSinceLastCommit();
                                 physicalRowsAdded += physicallyWrittenRowsSinceLastCommit;
-                                metrics.addRowsWritten(added, physicallyWrittenRowsSinceLastCommit, timeDelta);
+                                metrics.addApplyRowsWritten(added, physicallyWrittenRowsSinceLastCommit);
```

Comment:
How is it possible to calculate row application rate per second if  `timeDelta` is not tracked?

Reply:
Prometheus (and Grafana) would use scrape event timestamps to track the time. Yes, WAL application job may be sitting idle most of the time and spend only a few seconds within the given hour on actual work, but for monitoring that doesn't matter much. It's more important to be able to tell that WAL application keeps up with WAL writers or not. If we're interested in WAL application job performance, we could expose a cumulative timer metric (a counter) that would contain total time spent on WAL application. WDYT?

Reply:
Measuring job performance is important, it can show if we improve or regress in the future. It is also not skewed with number of parallel jobs we run while rate of rows per wall clock will be skewed.

We can track the application vs write row count to see if the job keeps up but we need to reason why it doesn't keep up and at what write patterns.

Time of transaction in the queue again doesn't not help to understand why it happens. Apply job can run full speed be very quick but still be behind because no-feedback writing can simply be faster. 

The way I see we should start worry about is when the apply rate starts to go down significantly. Especially if the physical row apply rate start to go down.

Reply:
So, do you like the idea of adding the "total time in WAL application" metric?

Reply:
I like existing row apply rate per second.

Reply:
OK, let me keep it as is then, although it doesn't follow [best practices](https://prometheus.io/docs/practices/naming/) for Prometheus metrics.

Reply:
Reverted the metric in 7b9db714052cff65a1971b211e712f6844d6b405